spec/features/form/form_navigation_spec.rb

```diff
@@ -89,4 +110,57 @@
       end
     end
   end
+
+  describe "Editing a log" do
+    it "a question page has a link allowing you to cancel your input and return to the check answers page" do
+      visit("logs/#{id}/tenant-code-test?referrer=check_answers")
+      click_link(text: "Cancel")
+      expect(page).to have_current_path("/logs/#{id}/setup/check-answers")
+    end
+
+    context "when clicking save and continue on a mandatory question with no input" do
+      let(:id) { empty_case_log.id }
+
+      it "shows a validation error on radio questions" do
+        visit("/logs/#{id}/renewal")
+        click_button("Save and continue")
+        expect(page).to have_selector("#error-summary-title")
+        expect(page).to have_selector("#case-log-renewal-error")
+        expect(page).to have_title("Error")
```

Comment:
Given that these lines are the same as those on 135-137 and 146-148 (minus a few minor changes) you could probably extract a method for this